data/states/keno/main.py

```diff
@@ -49,9 +49,33 @@ def __init__(self, casino_player):
         self.label.rect.center = self.rect.center
         self.color = '#181818'
         self.casino_player = casino_player
-        self.bet = 0
-        self.is_paid = False
-        self.winnings = 0
+        self._bet = 0
+        self._is_paid = False
+        self._winnings = 0
+
+    @property
```

Comment:
Ah. That part is temporary so I can create a backend code file that is not gui aware. This would allow the getters and setters to assign to the backend instead of having to immediately rewrite everywhere the value is referenced.